web3-dev/smart-contracts/getting-started.rst

```diff
@@ -84,117 +86,141 @@ Note that a `deployer.wasm` file has also been generated. It will be used to dep
 Deploy your smart contract
 --------------------------
 
-Your smart contract is now ready to be pushed and executed on the Massa network.
-In order to deploy it, you need to own a Massa wallet and some MAS coins on it.
+Your smart contract is now ready to be deployed and later executed on the Massa network.
+
+Uploading a smart contract on the Massa blockchains requires coins. On top of the usual gas
+costs, Massa has also :ref:`storage costs <storage-cost>`. In order to deploy your smart contract,
```

Comment:
I got the error 

`/Users/nathanmassa/dev/massa-docs/web3-dev/smart-contracts/getting-started.rst:91: WARNING: undefined label: 'storage-cost'`